src/monitor.cc

```diff
@@ -271,67 +299,57 @@ void NodeMonitor::setStatistics() {
 
     // milliseconds
     long timeDelta = (cur_time.tv_sec * 1000 + cur_time.tv_usec / 1000)
-        - (instance_->stats_.lastTime_.tv_sec * 1000 + instance_->stats_.lastTime_.tv_usec / 1000);
+        - (stats_.lastTime_.tv_sec * 1000 + stats_.lastTime_.tv_usec / 1000);
 
     // Update the number of requests processed
     // and the ratio CPU/req.
-    instance_->stats_.lastRequests_ = currReqs;
-    instance_->stats_.lastCpuPerReq_ = (reqDelta <= 0) ? 0 : (scpu + ucpu) / reqDelta;
-    instance_->stats_.lastJiffiesPerReq_ = (reqDelta <= 0) ? 0
+    stats_.lastRequests_ = currReqs;
+    stats_.lastCpuPerReq_ = (reqDelta <= 0) ? 0 : (scpu + ucpu) / reqDelta;
+    stats_.lastJiffiesPerReq_ = (reqDelta <= 0) ? 0
         : ((float) (sticks + uticks)) / reqDelta;
 
     // Request delta - requests since last check.
-    instance_->stats_.lastReqDelta_ = reqDelta;
-    instance_->stats_.timeDelta_ = timeDelta;
+    stats_.lastReqDelta_ = reqDelta;
+    stats_.timeDelta_ = timeDelta;
```

Comment:
even better, timeDelta_ in that stats_ struct is never used (other than being set here), so I'm just going to delete it.